availability-store/src/store.rs

```diff
@@ -60,33 +60,31 @@ fn candidate_key(candidate_hash: &Hash) -> Vec<u8> {
 	(candidate_hash, 2i8).encode()
 }
 
-fn available_chunks_key(relay_parent: &Hash, erasure_root: &Hash) -> Vec<u8> {
-	(relay_parent, erasure_root, 3i8).encode()
-}
-
 fn candidates_with_relay_parent_key(relay_block: &Hash) -> Vec<u8> {
 	(relay_block, 4i8).encode()
 }
 
 // meta keys
-fn awaited_chunks_key() -> [u8; 14] {
-	*b"awaited_chunks"
-}
+const AWAITED_CHUNKS_KEY: [u8; 14] = *b"awaited_chunks";
 
 fn validator_index_and_n_validators_key(relay_parent: &Hash) -> Vec<u8> {
 	(relay_parent, 1i8).encode()
 }
 
+fn available_chunks_key(candidate_hash: &Hash) -> Vec<u8> {
+	(candidate_hash, 2i8).encode()
```

Comment:
We could probably turn hese constants into an `enum` with explicit values? So we at least use named entities and may be sure that the values won't be used twice in two different places?

Reply:
Yep, seems beyond the scope of the PR though. There are lots of other refactorings and weirdness in the availability store to deal with.